<launch>
    <node pkg="pr2_mtc" name="docking" type="docking_node" output="screen">
        <remap from="cmd_vel" to="navigation/cmd_vel" />
        <param name="goal_tolerance" value="0.05" />
        <param name="max_speed" value="0.2" />
        <param name="P" value="2.0" />
        <param name="I" value="0.3" />
        <param name="max_integral" value="10" />
        <param name="control_period" value="0.1" />
    </node>
</launch>